### ISSUE TITLE
Do not display cancelled courses

### ISSUE THREAD

**Issue by tobiasbork** [corona-school/web-user-app#38]
Canceled courses should not be displayed even if I am the creator of the course.

**Comment by reinerschristopher** [comment]
@tobiasbork currently this feature might seem relevant since a lot of trashy courses where created due to the fact that the edit option wasn't available for some weeks. I would rather clean the DB of those "trash" courses. From a UX perspective it's good to see the cancelled courses, especially for participants, but also for instructors. 

To not show too many entries in the dashboard I would suggest not showing courses which ended 1 month ago.

**Comment by tobiasbork** [comment]
Fair enough. However, I would also show courses which ended more than one month ago. A section "Vergangene und Abgesagte Kurse" might make sense in this case.
